client/packages/requisitions/src/ResponseRequisition/DetailView/columns.ts

```diff
@@ -4,84 +4,96 @@ import {
-          accessor: ({ rowData }) => rowData.item.unitName,
-          getSortValue: rowData => rowData.item.unitName ?? '',
-        },
-      ],
-      [
-        'stockOnHand',
-        {
-          accessor: ({ rowData }) => rowData.itemStats.availableStockOnHand,
-          getSortValue: rowData => rowData.itemStats.availableStockOnHand,
-          label: 'label.our-soh',
-          description: 'description.our-soh',
-        },
-      ],
+      'itemCode',
       {
-        key: 'customerStockOnHand',
-        accessor: ({ rowData }) =>
-          rowData.linkedRequisitionLine?.itemStats?.availableStockOnHand,
-        getSortValue: rowData =>
-          rowData.linkedRequisitionLine?.itemStats?.availableStockOnHand ?? '',
-
-        label: 'label.customer-soh',
-        description: 'description.customer-soh',
-        width: 100,
-        align: ColumnAlign.Right,
+        accessor: ({ rowData }) => rowData.item.code,
+        getSortValue: rowData => rowData.item.code,
       },
-      [
-        'requestedQuantity',
-        { getSortValue: rowData => rowData.requestedQuantity },
-      ],
-      // TODO: Should only be visible if linked requisition has approvalStatus != None)
+    ],
+    [
+      'itemName',
       {
-        key: 'approvedQuantity',
-        label: 'label.approved-quantity',
+        accessor: ({ rowData }) => rowData.item.name,
+        getSortValue: rowData => rowData.item.name,
       },
+    ],
+    [
+      'itemUnit',
       {
-        key: 'approvalComment',
-        label: 'label.approval-comment',
+        accessor: ({ rowData }) => rowData.item.unitName,
+        getSortValue: rowData => rowData.item.unitName ?? '',
       },
-      ['supplyQuantity', { getSortValue: rowData => rowData.supplyQuantity }],
+    ],
+    [
+      'stockOnHand',
       {
-        label: 'label.remaining-to-supply',
-        description: 'description.remaining-to-supply',
-        key: 'remainingToSupply',
-        width: 100,
-        align: ColumnAlign.Right,
-        accessor: ({ rowData }) => rowData.remainingQuantityToSupply,
-        getSortValue: rowData => rowData.remainingQuantityToSupply,
+        accessor: ({ rowData }) => rowData.itemStats.availableStockOnHand,
+        getSortValue: rowData => rowData.itemStats.availableStockOnHand,
+        label: 'label.our-soh',
+        description: 'description.our-soh',
       },
-      GenericColumnKey.Selection,
     ],
+    {
+      key: 'customerStockOnHand',
+      accessor: ({ rowData }) =>
+        rowData.linkedRequisitionLine?.itemStats?.availableStockOnHand,
+      getSortValue: rowData =>
+        rowData.linkedRequisitionLine?.itemStats?.availableStockOnHand ?? '',
+
+      label: 'label.customer-soh',
+      description: 'description.customer-soh',
+      width: 100,
+      align: ColumnAlign.Right,
+    },
+    [
+      'requestedQuantity',
+      { getSortValue: rowData => rowData.requestedQuantity },
+    ],
+  ];
+
+  if (authoriseCustomerRequisitions) {
```

Comment:
I think you would want to use something similar to RequestRequisition `useIsRemoteAuthorisation`, rather then just the preference (otherwise could have there columns on requisitions that don't need authorisation, like non program requisition or where customer doesn't have the preference set)

Reply:
confused! I remember doing that and can't find it now.. will do the conflicts and come back to this.

Reply:
Ok, got it now. ta